lib/worker/form.js

```diff
@@ -14,7 +14,7 @@ const pushDraftToEnketo = ({ Forms }, event) =>
   Forms.getByActeeIdForUpdate(event.acteeId, undefined, Form.DraftVersion)
     .then((maybeForm) => maybeForm.map((form) => {
       // if there was no draft or this form isn't the draft anymore just bail.
-      if ((form.def.id == null) || (form.draftDefId !== form.def.id)) return;
+      if ((form.def.id == null) || !form.isCurrentDraft()) return;
```

Comment:
```suggestion
      if (!form.isCurrentDraft()) return;
```

If the suggested change is made from the other comment above, I think the first condition could be removed.